packages/flutter/test/widgets/keep_alive_test.dart

```diff
@@ -202,7 +202,7 @@ void main() {
     // The important lines below are the ones marked with "<----"
     expect(tester.binding.renderView.toStringDeep(minLevel: DiagnosticLevel.info), equalsIgnoringHashCodes(
       'RenderView#00000\n'
-      ' │ debug mode enabled - ${Platform.operatingSystem}\n'
+      ' │ debug mode enabled - linux\n'
```

Comment:
Can you explain the cause of this change? Was it wrong to not having broken before?

Reply:
No, it's just a cut/paste error when updating the golden string here.  Fixed.